lib/tasks/publishing_api.rake

```diff
@@ -47,4 +47,25 @@ namespace :publishing_api do
 
     puts "Finished queuing items for Publishing API"
   end
+
+  desc "Send publishable item links of a specific type to Publishing API (ie, 'CaseStudy')."
+  task :publishing_api_patch_links_by_type, [:document_type] => :environment do |_, args|
+    document_type = args[:document_type]
+    editions_id = Document.where(document_type: document_type)
```

Comment:
I've updated it in the exact moment you left the comment. It's different from your approach though, what do you think?